apps/api/src/app/controllers/tag.controller.ts

```diff
@@ -20,7 +17,7 @@ export const TAG_INDEX_ROUTE = 'tags';
 @ApiTags('tags')
 @Controller(TAG_INDEX_ROUTE)
 export class TagController {
-    constructor(private readonly repositoryProvider: RepositoryProvider) {}
+    constructor(private readonly tagQueryService: TagQueryService) {}
 
     @Get('/:id')
     async fetchById(@Res() res, @Param('id') id: string) {
```

Comment:
We should look into:|
1. Typing the `Res` \ `Req` objects
2. If there is a NestJS pattern that allows us to omit them while still wrapping our custom error handling